fixtures/uking/mini-spider-light.json

```
@@ -0,0 +1,317 @@
+        },
+        {
+          "dmxRange": [157, 166],
+          "type": "Effect",
+          "effectName": "Effect 16"
+        },
+        {
+          "dmxRange": [167, 176],
+          "type": "Effect",
+          "effectName": "Effect 17"
+        },
+        {
+          "dmxRange": [177, 186],
+          "type": "Effect",
+          "effectName": "Effect 18"
+        },
+        {
+          "dmxRange": [187, 196],
+          "type": "Effect",
+          "effectName": "Effect 19"
+        },
+        {
+          "dmxRange": [197, 206],
+          "type": "Effect",
+          "effectName": "Effect 20"
+        },
+        {
+          "dmxRange": [207, 216],
+          "type": "Effect",
+          "effectName": "Effect 21"
+        },
+        {
+          "dmxRange": [217, 226],
+          "type": "Effect",
+          "effectName": "Effect 22"
+        },
+        {
+          "dmxRange": [227, 236],
+          "type": "Effect",
+          "effectName": "Effect 23"
+        },
+        {
+          "dmxRange": [237, 246],
+          "type": "Effect",
+          "effectName": "Auto Mode"
+        },
+        {
+          "dmxRange": [247, 255],
+          "type": "Effect",
+          "effectName": "Sound Active Mode",
+          "soundControlled": true
+        }
+      ]
+    },
+    "Effect Speed": {
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Reset": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 240],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [241, 255],
+          "type": "Maintenance",
+          "hold": "5s",
+          "comment": "For 5 Seconds to reset"
+        }
+      ]
+    },
+    "LED 1": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "comment": "Red 1"
+      }
+    },
+    "LED 2": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "comment": "Green 1"
+      }
+    },
+    "LED 3": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "comment": "Blue 1"
+      }
+    },
+    "LED 4": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White",
+        "comment": "White 1"
+      }
+    },
+    "LED 5": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "comment": "Red 2"
+      }
+    },
+    "LED 6": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "comment": "Green 2"
+      }
+    },
+    "LED 7": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "comment": "Blue 2"
+      }
+    },
+    "LED 8": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White",
+        "comment": "White 2"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "7ch",
```

To follow our convention:

```suggestion
      "name": "7-channel",
      "shortName": "7ch",
```